Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -41,6 +41,15 @@ open class Player: AVPlayerViewController {
         }
     }
 
+    override open var preferredFocusEnvironments: [UIFocusEnvironment] {
+        let subviews = contentOverlayView?.subviews.first?.subviews
+        if let button = subviews?.first(where: { $0 is UIButton }), button.alpha > 0.1 {
```

Comment:
I consider that relying only on the visual element type (`UIButton`) to choose the focus is not safe (only works on the current UI implementation and plugins).
Can we pass a UI element identification on the new event (`requestFocusUpdate`) ?

Reply:
We changed it to a generic UIView and receives the UIView reference on the event userInfo, can you check again?